packages/terra-search-field/src/SearchField.jsx

```diff
@@ -250,8 +263,13 @@ class SearchField extends React.Component {
       customProps.className,
     );
 
-    const inputText = inputAttributes && Object.prototype.hasOwnProperty.call(inputAttributes, 'aria-label') ? inputAttributes['aria-label'] : intl.formatMessage({ id: 'Terra.searchField.search' });
+    const labelClassNames = cx([
```

Comment:
You don't really need a separate variable for this.  I would provide the classNames inline.